bark/models/observer/observer_model.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 fortiss GmbH
```

Comment:
wir könnten eigentlich mal auf 2021 gehen

Reply:
copy n paste; )

Reply:
Yep, but I think we should do this in a separate PR as this would cloud this PR and make it difficult to review